site/en/articles/reporting-api-migration/index.md

```diff
@@ -0,0 +1,483 @@
+---
+layout: 'layouts/blog-post.njk'
```

Comment:
Can we move this one to the blog? It's essentially news. So blog/reporting-api-migration/